lib/request-handler.spec.js

```diff
@@ -32,8 +30,6 @@ describe('The request handler', function() {
   before(analytics.load)
 
   let camp
-  const initialBadgeMaxAge = process.env.BADGE_MAX_AGE_SECONDS
```

Comment:
These tests now invoke `handleRequest` with different config instead of a modified environment.